src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -173,11 +172,9 @@ protected function failedAuthorization()
      */
     public function validated()
     {
-        $rules = $this->container->call([$this, 'rules']);
```

Comment:
This is a major breaking change as I know that this has been used on a lot of projects that I've worked on. Is this even really needed to fix the issue in question?